devel/docker-compose-plugin/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/docker/compose 2.32.1 v
+go.setup            github.com/docker/compose 2.36.2 v
 # Delete this on next update to use golang PortGroup's default ('archive')
 github.tarball_from tarball
```

Comment:
please act upon the comment here to remove this line. That will require to update the `checksums`, of course after cleaning the port first (i.e., `sudo port clean --all docker-compose-plugin`).